docs/tutorials/longer-metrics-storage.md

```diff
@@ -69,15 +67,16 @@ aren't ready to make the move.
 
 ## Using the round-robin database
 
-By default, Netdata uses a round-robin database to store 1 hour of per-second metrics. Here's the default setting for
-`history` in the `netdata.conf` file that comes pre-installed with Netdata.
+In previous versions, Netdata used a round-robin database to store 1 hour of per-second metrics. To see if you're still
```

Comment:
When we install netdata using netdata-static64.sh it creates a ``` nedata.conf``` with the following data:

```
# netdata can generate its own config which is available at 'http://<netdata_ip>/netdata.conf'
# You can download it with command like: 'wget -O /usr/local/netdata/etc/netdata/netdata.conf http://localhost:19999/netdata.conf'
```

A suggetion, instead to say for the person see the netdata.conf, it is safest to run the following command:

```
grep "memory mode" /var/log/netdata/error.log  | grep "INFO  : MAIN"
```

that will bring an output like this:

```
2019-10-03 13:30:29: netdata INFO  : MAIN : Host 'ceres' (at registry as 'hades') with guid 'YYYY' initialized, os 'linux', timezone 'UTC', tags '', program_name 'netdata', program_version 'v1.17.1-164-ged371b15', update every 1, memory mode ram, history entries 3996, streaming disabled (to '' with api key ''), health enabled, cache_dir '/var/cache/netdata', varlib_dir '/var/lib/netdata', health_log '/var/lib/netdata/health/health-log.db', alarms default handler '/usr/libexec/netdata/plugins.d/alarm-notify.sh', alarms default recipient 'root'

Reply:
I'm a little confused about how this is simpler than asking them to open up `netdata.conf`. Does the command offer better information, or are you thinking it's easier to pop that command into the terminal and look at the output?

Reply:
> grep "memory mode" /var/log/netdata/error.log  | grep "INFO  : MAIN"


there is no guarantee the path is `/var/log/...`

Reply:
I used this, because it is the default option when we do not use `--install` and I found error.log generic.

Reply:
ok, same goes for "memory mode", dont forget logs are rotated

Reply:
I was thinking here, I believe the safest solution will be:

```
curl -o netdata_chart_list.json -k https://localhost:19999/api/v1/charts?all ; grep memory_mode netdata_chart_list.json
```

The expected result is

![report](https://user-images.githubusercontent.com/49162938/66394667-3c3d7300-e9c5-11e9-872c-d9415f974064.png)